apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/Util.kt

```diff
@@ -12,8 +12,17 @@ private val JAVA_RESERVED_WORDS = arrayOf(
     "transient", "try", "true", "void", "volatile", "while"
 )
 
+private val KOTLIN_RESERVED_WORDS = arrayOf(
```

Comment:
I think you are missing a lot of words from https://kotlinlang.org/docs/reference/keyword-reference.html  For example  I don't see `suspend`

Reply:
Well there are 2 kind of keywords, hard and soft. This array is hard keywords, usage soft keywords depends on the context. Maybe later we need to expand this list.